app/static/src/scss/includes/_table.scss

```diff
@@ -1,4 +1,11 @@
 .govuk-table {
+  border-bottom: 1px solid #b1b4b6;
```

Comment:
what does this fix?

Reply:
@ltrasca the previous PR was applying these styles to just the browse page table, this applies it to all of them

Reply:
sweet 👌 . I guess the visual regression tests need updating. Theyre still not running in ci though, so will leave it up to you